src/functions/uploadToRSIS/application/data-mapper/__tests__/cat-c-mapper.spec.ts

```diff
@@ -0,0 +1,248 @@
+      { col: 'JUNCTIONS_TURN_RIGHT_TOTAL', val: 29 },
+      { col: 'JUNCTIONS_TURN_LEFT_TOTAL', val: 30 },
+      { col: 'JUNCTIONS_TURN_CUT_TOTAL', val: 31 },
+      { col: 'JUDGEMENT_OVER_TOTAL', val: 32 },
+      { col: 'JUDGEMENT_MEET_TOTAL', val: 33 },
+      { col: 'JUDGEMENT_CROSS_TOTAL', val: 34 },
+      { col: 'POSITIONING_NORMAL_TOTAL', val: 35 },
+      { col: 'POSITIONING_LANE_TOTAL', val: 36 },
+      { col: 'PEDESTRIAN_CROSSING_TOTAL', val: 37 },
+      { col: 'POSTITION_STOPS_TOTAL', val: 38 },
+      { col: 'AWARENESS_PLAN_TOTAL', val: 39 },
+      { col: 'ANCILLARY_CONTROLS_TOTAL', val: 40 },
+      { col: 'EYESIGHT_SERIOUS', val: 0 },
+      { col: 'REV_LEFT_TRAIL_CONT_SERIOUS', val: 0 },
+      { col: 'REV_LEFT_TRAIL_OBSERV_SERIOUS', val: 0 },
+      { col: 'PRECAUTIONS_SERIOUS', val: 0 },
+      { col: 'CONTROL_ACC_SERIOUS', val: 0 },
+      { col: 'CONTROL_CLUTCH_SERIOUS', val: 0 },
+      { col: 'CONTROL_GEARS_SERIOUS', val: 0 },
+      { col: 'CONTROL_FOOTBRAKE_SERIOUS', val: 0 },
+      { col: 'CONTROL_PARK_SERIOUS', val: 0 },
+      { col: 'CONTROL_STEERING_SERIOUS', val: 0 },
+      { col: 'MOVE_OFF_SAFETY_SERIOUS', val: 0 },
+      { col: 'MOVE_OFF_CONTROL_SERIOUS', val: 0 },
+      { col: 'MIRRORS_MC_REAR_SIG_SERIOUS', val: 0 },
+      { col: 'MIRRORS_MC_REAR_DIR_SERIOUS', val: 0 },
+      { col: 'MIRRORS_MC_REAR_SPE_SERIOUS', val: 0 },
+      { col: 'SIGNALS_NECESSARY_SERIOUS', val: 0 },
+      { col: 'SIGNALS_CORRECTLY_SERIOUS', val: 0 },
+      { col: 'SIGNALS_TIMED_SERIOUS', val: 0 },
+      { col: 'CLEARANCE_OBSTRUCT_SERIOUS', val: 0 },
+      { col: 'RESPONSE_TRAF_SIGNS_SERIOUS', val: 0 },
+      { col: 'RESPONSE_ROAD_MARK_SERIOUS', val: 0 },
+      { col: 'RESPONSE_TRAF_LIGHT_SERIOUS', val: 0 },
+      { col: 'RESPONSE_TRAF_CONT_SERIOUS', val: 0 },
+      { col: 'RESPONSE_OTHER_SERIOUS', val: 0 },
+      { col: 'USE_OF_SPEED_SERIOUS', val: 0 },
+      { col: 'FOLLOWING_DISTANCE_SERIOUS', val: 0 },
+      { col: 'MAINTAIN_PROG_SPEED_SERIOUS', val: 0 },
+      { col: 'MAINTAIN_PROG_HES_SERIOUS', val: 0 },
+      { col: 'JUNCTIONS_SPEED_SERIOUS', val: 0 },
+      { col: 'JUNCTIONS_OBSERV_SERIOUS', val: 0 },
+      { col: 'JUNCTIONS_TURN_RIGHT_SERIOUS', val: 0 },
+      { col: 'JUNCTIONS_TURN_LEFT_SERIOUS', val: 0 },
+      { col: 'JUNCTIONS_TURN_CUT_SERIOUS', val: 0 },
+      { col: 'JUDGEMENT_OVER_SERIOUS', val: 0 },
+      { col: 'JUDGEMENT_MEET_SERIOUS', val: 0 },
+      { col: 'JUDGEMENT_CROSS_SERIOUS', val: 0 },
+      { col: 'POSITIONING_NORMAL_SERIOUS', val: 0 },
+      { col: 'POSITIONING_LANE_SERIOUS', val: 0 },
+      { col: 'PEDESTRIAN_CROSSING_SERIOUS', val: 0 },
+      { col: 'POSTITION_STOPS_SERIOUS', val: 0 },
+      { col: 'AWARENESS_PLAN_SERIOUS', val: 0 },
+      { col: 'ANCILLARY_CONTROLS_SERIOUS', val: 0 },
+      { col: 'REV_LEFT_TRAIL_CONT_DANGEROUS', val: 0 },
+      { col: 'REV_LEFT_TRAIL_OBSER_DANGEROUS', val: 0 },
+    //   { col: 'CONTROL_STOP_PROMPT_DANGEROUS', val: 0 },
```

Comment:
Can we remove these or are they needed?

Reply:
Copied from existing code but i'll clean them all out.